packages/wireshark.vm/tools/chocolateyinstall.ps1

```diff
@@ -7,9 +7,15 @@ try {
 
   $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
   $shortcut = Join-Path $shortcutDir "$toolName.lnk"
-  $executablePath = Join-Path ${Env:ProgramFiles} "Wireshark\Wireshark.exe" -Resolve
+  $executablePath = Join-Path ${Env:ProgramFiles} "Wireshark\$toolName.exe" -Resolve
   Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath -RunAsAdmin
   VM-Assert-Path $shortcut
+
+  # Add tshark (installed by Wireshark) to PATH and to Tools directory
+  $toolName = 'tshark'
+  $executablePath = Join-Path ${Env:ProgramFiles} "Wireshark\$toolName.exe" -Resolve
+  Install-BinFile -Name $toolname -Path $executablePath
+  VM-Install-Shortcut -toolName $toolName -category $category -executablePath $executablePath -consoleApp $true
```

Comment:
the shortcut executable is `tshark.exe` already starts capturing on the first interface, perhaps it would be easier to execute with `--help`?

Reply:
Good point! Any other opinions @mandiant/flare-vm?